shell/platform/tizen/external_texture_gl.cc

```diff
@@ -30,69 +30,37 @@ struct ExternalTextureGLState {
 
 static std::atomic_long nextTextureId = {1};
 
+static void UnmarkTbmSurfaceToUse(void* surface) {
+  FT_ASSERT(surface);
+  tbm_surface_h tbm_surface = (tbm_surface_h)surface;
+  tbm_surface_destroy(tbm_surface);
+}
+
+static void UnmarkMediaPacketToUse(void* surface) {
+  FT_ASSERT(surface);
+  media_packet_destroy((media_packet_h)surface);
```

Comment:
I think media_packet_ref should be called to prevent this handle from destroy. The same thing happen for tbm surface.

Reply:
I'm a little confusing... you think calling ``media_packet_destroy`` at here is problem? 
Is it correct that I understand? Why?

Reply:
I mean, if you call media_packet_destroy directly, that media handle may be really destroyed, media player need to allocate another media handle, but if you call media_packet_ref when mark and call media_packet_destroy when unmark, that handle won't be really destroyed, media player may reuse it later.

Reply:
Whatever External texture sources were used, it now seems only Tizen embedder is responsible for destroy them.
I think this is a good approach. Now we don't need to use ref/unref anymore.